documentation.yml

```diff
@@ -1,50 +1,50 @@
 toc:
   - name: DOM Interaction Helpers
-
-  - click
-  - tap
-  - focus
-  - blur
-  - triggerEvent
-  - triggerKeyEvent
-  - fillIn
+    children:
+      - click
+      - tap
+      - focus
+      - blur
+      - triggerEvent
+      - triggerKeyEvent
+      - fillIn
```

Comment:
I tried this at first, but it looked strange in the HTML version. Is this still the case?

Reply:
Yes, this _only_ works for the `md` version. If you do `children:` in the HTML formatted output its _really really_ weird.